### ISSUE TITLE
Wording issue?

### ISSUE THREAD

**Issue by lsetiawan** [oceanhackweek/cahw2018_tutorials#5]
Hi @friedrichknuth,

Sorry to open up this issue, but I think your wording [here](https://github.com/oceanhackweek/cahw2018_tutorials/blame/master/ooi_m2m_demo/ooi_m2m_data_access/README.md#L20) of "virtual environment" might be misleading. 

In python there is a concept of `virtualenv` which is a python virtual environment where it works parallel with `pip`. But you can't install Python packages from conda repository with it. And there's a concept of conda, which is a package **and** environment manager. So python packages from conda has to be installed within a **conda environment**.

Long story short, it might mislead people that already have some python background to say that you're creating a virtual environment, but rather a conda environment since you're installing python packages from conda. Thanks :smile: Sorry for the long message, small change but possible big confusion.

**Comment by lsetiawan** [comment]
I think @friedrichknuth has addressed this. Closing. Thanks!
